README.md

```diff
@@ -15,7 +15,6 @@ List of arguments that can be passed to the odf-must-gather are:
 ```
 -o,  --odf                Collect ODF logs
```

Comment:
Collect ODF logs (includes content from arguments n,c,cl,cs contents)

Reply:
Or, shall we write it as (includes everything except DR)?

Reply:
Right now, it makes no point to use `-o` in conjunction with anything other than `-dr`.

Reply:
we will have to update it again after we include other parameters, so better to clearly mention the contents.

Reply:
Ack